### PR TITLE
[FIX] Fixes for xenomorphs

### PR DESCRIPTION
**Pull request by ErdGinalD**
## What Does This PR Do

- Xenos now bleed with lime blood color, instead of red.
- Xenos now doesnt have neutral fraction, as any other biohazard(blod, spooder, demons). Now its only alien.

## Why It's Good For The Game

Red blood is not good for immersion
Neutral xenos are not exist in this world

## Images of changes
Neutral xenos - the only biohazard you have. This is clearly an oversight
image

Before update:
image

After fix:
image

## Testing

Spawn xeno, kill him. drag him, see the green blood as expected.
Spawn goliath, he attack xenos. Kill him, revive him with lazarus, now he is neutral but still attack xenos, as he do with spiders or blobs.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
fix: Xenomorph blood is green again
tweak: Neutral mobs are now hostile to xenomorphs, similar to other biohazards
/:cl:

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -661,6 +661,8 @@
 				var/mob/living/carbon/human/H = src
 				if(H.dna.species.blood_color)
 					existing_trail.color = H.dna.species.blood_color
+			else if(isalien(src))
+				existing_trail.color = "#00FF32"
```
Comment (Contrabang) [inline code/modules/mob/living/living.dm:665]: You should try with this color, this is the color used by other xeno blood stuff
```suggestion
				existing_trail.color = "#05EE05"
```

**Review by Contrabang — CHANGES_REQUESTED** [review]

**Review by Sirryan2002 — APPROVED** [review]

File: code/modules/mob/living/carbon/alien/alien_base.dm [hunk 2]
```diff
@@ -25,6 +25,7 @@
 /mob/living/carbon/alien/Initialize(mapload)
 	. = ..()
 	create_reagents(1000)
+	faction = list("alien")
```
Comment (DGamerL) [inline code/modules/mob/living/carbon/alien/alien_base.dm:28]: Please put this in the type declaration above

**Review by DGamerL — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/alien/alien_base.dm (hunk 2 above)
Comment (ErdGinalD) [reply code/modules/mob/living/carbon/alien/alien_base.dm]: Donezo!

**Review by DGamerL — APPROVED** [review]
Seems fine

**Review by Contrabang — APPROVED** [review]

**Pull request merged**